### DISCUSSION TITLE
the default e-mail configuration is not working

### DISCUSSION THREAD

**Discussion by abraralmadi** [globaleaks/globaleaks-whistleblowing-software#4547] [Q&A]
Hello,

I am configuring Globaleaks blow the whistle app but the default e-mail configuration is not working. no error but email is not reaching the end user :

 root@2669a72f5757:/# telnet mail.globaleaks.org 9267
Trying 51.15.224.196...
Connected to mail.globaleaks.org.
Escape character is '^]'.
220 DEMO GLOBALEAKS SYSTEM TO RELAY DEFAULT NOTIFICATIONS

Is firewall blocking something?

Thanks in advance.

**Comment by evilaliv3** [comment]
@abraralmadi : May i ask you which provider are you using and to check if they do not implement a default firewall that prevents SMTP connection?

Many providers infact blocks those kind of connections by default to get sure their network is not abusted by malware/spam.

**Comment by abraralmadi** [comment]
@evilaliv3 
i personally don't think it caused by the provider firewall because the telnet mail.globaleaks.org 9267 working fine

**Comment by abraralmadi** [comment]
this is the logs


2021-03-22T07:21:43+0000 [stdout#info] [D] [1] Sending email to MyEmail@email.com using SMTP server [mail.globaleaks.org:9267] [TLS]
2021-03-22T07:21:43+0000 [twisted.mail.smtp.ESMTPSenderFactory#info] Starting factory <twisted.mail.smtp.ESMTPSenderFactory object at 0x7f8704db30f0>
2021-03-22T07:21:44+0000 [twisted.python.log#info] WB.website.com private IP - - [22/Mar/2021:07:21:44 +0000] "PUT /admin/config HTTP/2" 202 0 1318ms - "Mozilla/5.0 (Macintosh; Intel Mac OS X 11_2_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.82 Safari/537.36" 1
2021-03-22T07:21:45+0000 [twisted.mail.smtp.ESMTPSenderFactory#info] Stopping factory <twisted.mail.smtp.ESMTPSenderFactory object at 0x7f8704db30f0>

**Comment by evilaliv3** [comment]
GlobaLeaks do not use anymore port 9267 in it's default configuration.

I suspect you may be using an old version installed on a possible old
software.

Would you please reset the smtp configuration and retry?

Which version of Globalwaks and which OS / version are you currently
running?

**Comment by abraralmadi** [comment]
Software version: 4.0.31
what does it use now what is the port?

**Comment by abraralmadi** [comment]
@evilaliv3 can you please help i can't update now i need to solve this issue temperer

**Comment by abraralmadi** [comment]
@evilaliv3 i try to clone the GlobaLeaks to create updated image as i used to but the docker file folder is not there any more can you Please help me

**Comment by abraralmadi** [comment]
i used to do these steps 

git clone <https://github.com/globaleaks/GlobaLeaks>
 
 
cd GlobaLeaks
 
cd docker
 
sudo docker build -t <globaleaks/globaleaks> .
 
sudo docker run --rm -it --network host -p '80:80' -p '443:443' <globaleaks/globaleaks>
 
docker login --username=your username
 
docker images
 
docker tag <imagename> < username /repository>:<tagname>
 
push < username /repository>:<tagname>

**Comment by abraralmadi** [comment]
because our environment only accept it in docker image format

**Comment by evilaliv3** [comment]
The docker packages are currently archived here to not be confused for
stable and production grade scripts. Can you use them from here? Do you
require any change for making them work? In this case we may consider
applying some patches.

https://github.com/globaleaks/globaleaks-docker-scripts

**Comment by evilaliv3** [comment]
Closing this ticket as probably solved. Feel free @abraralmadi to reoped this in case further support is needed on the email configuration.
